<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Window</string>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">

    <!-- txtName -->
    <item>
     <widget class="QLineEdit" name="txtName">
      <property name="placeholderText">
       <string>Lake name</string>
      </property>
     </widget>
    </item>

    <!-- spinTemp -->
    <item>
     <widget class="QSpinBox" name="spinTemp">
      <property name="minimum">
       <number>-30</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
    </item>

    <!-- spinVisitors -->
    <item>
     <widget class="QSpinBox" name="spinVisitors">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </item>

    <!-- Buttonleiste -->
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPrevious">
        <property name="text">
         <string>Previous</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNext">
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>

    <!-- Liste -->
    <item>
     <widget class="QListWidget" name="listWidgetEntries"/>
    </item>

   </layout>
  </widget>

  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>

  <widget class="QStatusBar" name="statusbar"/>
 </widget>

 <resources/>
 <connections/>
</ui>
